packages/bson-transpilers/test/yaml/bson.yaml

```
@@ -254,6 +284,16 @@ tests:
           csharp: Convert.ToInt32("3")
           object: new Int32('3')
           ruby: "'3'.to_i"
+          go: |-
+            var parseInt32 = func(str string) int32 {
+            i64, err := sstrconv.ParseInt(str, 10, 32)
```

```suggestion
            i64, err := strconv.ParseInt(str, 10, 32)
```
I think also missing an indent.